apps/ssr-tests-v9/src/build.ts

```diff
@@ -25,35 +25,37 @@ async function build() {
 
   const generateStartTime = process.hrtime();
 
-  // Portals currently do not support hydration
-  // https://github.com/facebook/react/issues/13097
-  const skippedPaths = ['react-portal'];
-
-  const rawStoriesGlobs = getPackageStoriesGlob({
+  const rawStoriesGlobs: string[] = getPackageStoriesGlob({
     packageName: '@fluentui/react-components',
     callerPath: __dirname,
-  }).filter(
-    (storyPath: string) =>
-      // only return entries that don't match any of the skippedPaths
-      !skippedPaths.find(skippedPath => {
-        if (storyPath.includes(skippedPath)) {
-          return true;
-        }
-        return false;
-      }),
-  ) as string[];
+  });
 
+  // Add stories defined in the package
   rawStoriesGlobs.push(path.resolve(path.join(__dirname, './stories/**/index.stories.tsx')));
+
   const storiesGlobs = rawStoriesGlobs
     // TODO: Find a better way for this. Pass the path via params? 👇
     .map(pattern => path.resolve(__dirname, pattern));
 
-  await generateEntryPoints({
+  const { ignoredStories } = await generateEntryPoints({
     esmEntryPoint,
     cjsEntryPoint,
     storiesGlobs,
+    ignore: [
```

Comment:
it would be awesome if we could implement json config for this without need to go to source code.

Reply:
Agreed, but I hope that it will not become a widely used pattern.